marklogic-client-api/src/test/java/com/marklogic/client/test/rows/VectorTest.java

```diff
@@ -108,4 +109,23 @@ void cosineSimilarity_InvalidVector() {
 		assertTrue(actualMessage.contains("arg2 is not of type vec:vector"), "Unexpected message: " + actualMessage);
 	}
 
+	@Test
+//	@Disabled("See https://progresssoftware.atlassian.net/browse/MLE-15707")
+	// As of 07/26/24, this test will fail with the 12.0.nightly-ubi-2.0.1 build.
+	// However, it will succeed with the 12ea1 build.
+	void bindVectorFromDocs() {
+		PlanBuilder.ModifyPlan plan =
+			op.fromSearchDocs(
+				op.cts.andQuery(
+					op.cts.documentQuery("/optic/vectors/alice.json"),
+					op.cts.elementQuery(
+						"person",
+						op.cts.andQuery()
+    				)
+  				))
+			.bind(op.as("embedding", op.vec.vector(op.xpath("doc", "/person/embedding"))))
+			.limit(5);
```

Comment:
Remove the limit call here, it's implying that it's needed but since the assertion expects a single document, it must not be.